### ISSUE TITLE
Cannot add repo with argocd repo add command

### ISSUE THREAD

**Issue by christianh814** [argoproj-labs/argocd-operator#55]
When I run the following command to add a repo

```
argocd repo add https://github.com/christianh814/gitops-examples
```

It does not actually add it. It seems like it's getting overwritten by the operator.

When I add a repo via the operator...

```
apiVersion: argoproj.io/v1alpha1
kind: ArgoCD
metadata:
  name: example-argocd
spec:
  repositories: |
    - url: https://github.com/christianh814/another-repo
```

It'll add the repo...but I still can't add additional repos via the CLI.

I would like to be able to add additional repos, not only ones managed by the operator.

**Comment by jmckind** [comment]
Thanks for reporting this @christianh814! This bug will also affect the SSH Known hosts as well.

**Comment by jmckind** [comment]
Released in v0.0.8
